django/db/backends/base/operations.py

```diff
@@ -517,12 +518,20 @@ def convert_durationfield_value(self, value, expression, context):
         return value
 
     def check_aggregate_support(self, aggregate_func):
-        """Check that the backend supports the provided aggregate
+        warnings.warn(
+            "check_aggregate_support has been deprecated. Use "
+            "check_expression_support instead.",
+            RemovedInDjango20Warning, stacklevel=2)
```

Comment:
The deprecation will need to wait for the `RemovedInDjango21Warning` to land on master before this is ready to go. I'll also add deprecation notes then.

As for the methods themselves, it should be fine to call check_aggregate from check_expression, and only deprecate check_aggregate from 1.9.